packages/react-composites/review/beta/react-composites.api.md

```diff
@@ -149,6 +149,8 @@ export interface CallAdapterCallManagement {
     createStreamView(remoteUserId?: string, options?: VideoStreamOptions): Promise<void | CreateVideoStreamViewResult>;
     disposeStreamView(remoteUserId?: string, options?: VideoStreamOptions): Promise<void>;
     // @beta
+    getEnvironmentInfo(): Promise<EnvironmentInfo>;
```

Comment:
Anything with a `get` should really be idempotent (i.e. have no side effects), because of our stateful design it populates state ... I wonder (at the stateful layer) if we shouldn't be calling this `getEnvironmentInfo` and instead something like populateEnvironmentInfo or fetchEnvironmentInfo to indicate there will be a side effect whereby the state gets populated (and thus further changes could propagate)

Reply:
cc @PorterNan @prprabhu-ms